tests/test_views.py

```diff
@@ -4140,6 +4145,30 @@ def test_creates_a_project(self):
         assert_true(node)
         assert_true(node.title, 'Im a real title')
 
+    def test_create_component_add_contributors(self):
+        url = web_url_for('project_new_node', pid=self.project._id)
+        post_data = {'title': 'New Component With Contributors Title',  'category': '', 'inheritContributors': 'true'}
+        request = self.app.post(url, post_data, auth=self.user1.auth)
```

Comment:
This variable name is misleading; `post` returns a response object, not a request. By convention, we use `res` for the return value of webtest requests.